po/da.po

```diff
@@ -1902,13 +1922,15 @@ msgstr ""
 
 #: src/welcome-day-1.md:1
 msgid "# Welcome to Day 1"
-msgstr ""
+msgstr "# Velkommen til Dag 1"
 
 #: src/welcome-day-1.md:3
 msgid ""
 "This is the first day of Comprehensive Rust. We will cover a lot of ground\n"
 "today:"
 msgstr ""
+"Dette er den første dag af Comprehensive Rust. Vi vil gå over en samme\n"
```

Comment:
Missing word here? Did you mean "Vi vil gå over en masse i dag"?

Reply:
Thanks you, definitely missed a word!

Reply:
```suggestion
"Dette er den første dag af Comprehensive Rust. Vi kommer til at dække en masse terræn\n"
```